test-harness/alembic/versions/2635ee8b59d5_deprecated_upgrade.py

```diff
@@ -0,0 +1,29 @@
+"""deprecated upgrade
+
+Revision ID: 2635ee8b59d5
+Revises: 773d67fb8a72
+Create Date: 2023-03-30 15:06:15.847773
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2635ee8b59d5'
+down_revision = '773d67fb8a72'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.sync_enum_values('public', 'simpleenum', ['A', 'B', 'C'], ['A', 'B', 'C', 'D'])
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    #op.sync_enum_values('public', 'simpleenum', ['A', 'B', 'C', 'D'], ['A', 'B', 'C'])
```

Comment:
@dw Excluding the v0.0.2 downgrade test since we didn't expect it to work under most postgres conditions.